### DISCUSSION TITLE
Alternate geometries?

### DISCUSSION THREAD

**Discussion by DKMiller71** [viresh-ratnakar/exolve#83] [Q&A] [answered]
Does Exolve have the ability to present alternate geometries, like hex grids or circular?

**Comment by viresh-ratnakar** [comment, marked as answer]
Not right now (well, 3-D is supported). I would like to add this. Do you
have a puzzle planned? If so, when?

I think I would generalize by allowing parameterized cells, with the
parameters determining the cell shape, location, connections, and canned
support for some standard ones like circular, spiral, hex, etc.

**Reply by DKMiller71** [reply]
I didn't have anything specifically planned, but had run across some alternate designs and was curious what Exolve could support if I were to play around with trying one of them

**Reply by DKMiller71** [reply]
For what it's worth, I never got back to you on this, but I had created a hex grid puzzle back in January.  Hex grids have two optional layouts (point up or flat side up, I did the former).  Here if you want to see it: 

https://darrenspuzzles.blogspot.com/2023/01/cryptic-snowflake-2023-01.html

If you ever do have support for hex grid, I'd probably add the fillable option.

**Reply by DKMiller71** [reply]
If you ever do decide to do this, let me know... I have a third on the way.

**Reply by viresh-ratnakar** [reply]
Had forgotten about this, and it still seems like a good feature to add! Please open a bug (I can do that too, if you can't get to it).

What do you mean by the "fillable option"?

**Reply by DKMiller71** [reply]
> What do you mean by the "fillable option"?

I just meant that: right now, I've only got the PDF version published, and if Exolve were able to support hex grids, I'd put the Exolve version out there.
